spec/rails_helper.rb

```diff
@@ -33,7 +33,7 @@
 
 FactoryBot.find_definitions
 
-WebMock.disable_net_connect!(allow_localhost: true, allow: /^elasticsearch[-a-z0-9]*$/)
+WebMock.disable_net_connect!(allow_localhost: true, allow: %r{^https?://elasticsearch[-a-z0-9]*(?::\d+)?/})
```

Comment:
```suggestion
WebMock.disable_net_connect!(allow_localhost: true, allow: %r{.*elasticsearch.*})
```

Minor: something simpler should be suitable for testing, noting it's unclear that we'll be making other, stubbed calls to anything with "elasticsearch" in the URL.

Reply:
My only concern with using a general regex like that is it would allow running tests against a live database if it happened to have elasticsearch in the url, however that risk is relatively low so happy to change it if you want.